DIPs/DIP1003.md

```diff
@@ -0,0 +1,239 @@
+1. Do not implement the suggested changes and do not allow `body` as a symbol name                                                                                                    | No          | No                            | No          | n/a                          | n/a                          |
+2. Replace `body` with `function` in the next release of D                                                                                                                            | Yes         | No                            | No          | Difficult                    | Simple                       | Not feasible as it abruptly breaks user code with no warning
+3. Remove `body` in the next release of D and disallow its use to denote the function body. Instead, accept an un-annotated block and allow `body` as a symbol name                   | Yes         | No                            | No          | Difficult                    | Simple                       | Not feasible as it abruptly breaks user code with no warning
+4. Keep `body` as a conditional keyword and allow it as a sybol name in all other contexts                                                                                            | No          | No                            | Yes         | Easy                         | Complex                      | Conditional keywords are opposed by Walter Bright
+5. Add `function` as an optional keyword in the place of `body` and schedule the `body` keyword for deprecation. After this period allow `body` as a symbol name                      | Yes         | Yes                           | Yes         | Moderate                     | Moderate                     |
+6. Allow the function body to optionally be annotated with `body`, but do not require it. Schedule the `body` keyword for deprecation. After this period, allow body as a symbol name | Yes         | Yes                           | Yes         | Moderate                     | Moderate                     | Makes function contracts 1 line shorter
+7. Schedule `body` for deprecation and then removal. After this period, add `function` in its place and allow `body` as a symbol name                                                 | Yes         | Yes                           | No          | Difficult                    | Moderate                     |
+8. Schedule `body` for deprecation and then removal. After this period, disallow its use to denote the function body and allow `body` as a symbol name                                | Yes         | Yes                           | No          | Difficult                    | Moderate                     | Makes function contracts 1 line shorter
```

Comment:
I've only included those options for completeness' sake. Later on I specifiy that these are not recommended.

I'll look at splitting it up as you suggested.